cf/cellconnectivity.py

```diff
@@ -0,0 +1,185 @@
+
+    In CF-netCDF a cell topology construct can only be provided by a
+    UGRID mesh topology variable. The construct array is supplied
+    either indirectly by any of the UGRID variables that are used to
+    define a domain topology construct, or directly by the UGRID
+    "face_face_connectivity" variable (for face cells). In the direct
+    case, the integer indices contained in the UGRID variable may be
+    used as the cell identities, although the CF data model attaches
+    no significance to the values other than the fact that some values
+    are the same as others.
+
+    Restricting the types of connectivity to those implied by the
+    geospatial topology of the cells precludes connectivity derived
+    from any other sources, but is consistent with UGRID encoding
+    within CF-netCDF.
+
+    See CF Appendix I "The CF Data Model".
+
+    **NetCDF interface**
+
+    {{netCDF variable}}
+
+    .. versionadded:: 3.16.0
+
+    """
+
+    @property
+    def connectivity(self):
+        """The connectivity type.
+
+        {{cell connectivity type}}
+
+        .. versionadded:: 3.16.0
+
+        .. seealso:: `del_connectivity`, `get_connectivity`,
+                     `has_connectivity`, `set_connectivity`
+
+        """
+        return self.get_connectivity(default=AttributeError())
+
+    @connectivity.setter
+    def connectivity(self, value):
+        self.set_connectivity(value)
+
+    @connectivity.deleter
+    def connectivity(self):
+        self.del_connectivity(default=AttributeError())
+
+    def identity(
+        self,
+        default="",
+        strict=None,
+        relaxed=False,
+        nc_only=False,
+        relaxed_identity=None,
+    ):
+        """Return the canonical identity.
+
+        By default the identity is the first found of the following:
+
+        * The connectivity type type, preceded by ``'connectivity:'``.
+        * The `standard_name` property.
+        * The `id` attribute, preceded by ``'id%'``.
+        * The `long_name` property, preceded by ``'long_name='``.
+        * The netCDF variable name, preceded by ``'ncvar%'``.
+        * The value of the *default* parameter.
+
+        .. versionadded:: 3.16.0
+
+        .. seealso:: `id`, `identities`, `long_name`, `connectivity`,
+                     `nc_get_variable`, `standard_name`
+
+        :Parameters:
+
+            default: optional
+                If no identity can be found then return the value of the
+                default parameter.
+
+            strict: `bool`, optional
+                If True then the identity is the first found of only
+                the `connectivity` attribute, "standard_name" property
+                or the "id" attribute.
+
+            relaxed: `bool`, optional
+                If True then the identity is the first found of only
+                the `connectivity `attribute, the "standard_name"
+                property, the "id" attribute, the "long_name" property
```

Comment:
```suggestion
            strict: `bool`, optional
                If True then the identity is the first found of only
                the "connectivity" attribute, "standard_name" property
                or the "id" attribute.

            relaxed: `bool`, optional
                If True then the identity is the first found of only
                the "connectivity" attribute, the "standard_name"
                property, the "id" attribute, the "long_name" property
```

just to be consistent across the descriptions (alternatively you could convert them all to be quoted with backticks).